.travis.yml

```diff
@@ -8,6 +8,7 @@ before_script:
   - cp config/database.yml.travis config/database.yml
   - bundle exec rake db:setup
 addons:
+  postgresql: "9.5"
```

Comment:
without this, the rake task wouldn't be able to compute sql commands?

Reply:
@jlopker Yes. I've added a jsonb datatype which doesn't exist in Postgresql 9.1